requirements/base.txt

```diff
@@ -37,7 +37,7 @@ click-repl==0.2.0
     # via celery
 code-annotations==1.2.0
     # via edx-toggles
-cryptography==35.0.0
+cryptography==36.0.0
```

Comment:
Ony one incompatible change mentioned in the [changelog](https://github.com/pyca/cryptography/blob/main/CHANGELOG.rst#3600---2021-11-21) which shouldn't be an issue.